services/cd-service/pkg/valid/validations.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	MaxAppNameLen  = 39
+	MaxAppNameLen  = 70
```

Comment:
also we have to check if this breaks the UI layout in any way.

Reply:
I tested it with the application name `myappiswaytoolongdontyouthinkisnitormaybeitisnotmaybewejusthaveareally`


![image](https://github.com/freiheit-com/kuberpult/assets/133217951/6c1d9d76-ac06-4129-b20f-07591a7bc2eb)

Reply:
How does it look when adding an action (like deploy or lock) to the planned actions? This should also render nice.

Reply:
![image](https://github.com/freiheit-com/kuberpult/assets/133217951/e977d8e7-f779-4d1d-a0e4-c0d8914bb8da)
![image](https://github.com/freiheit-com/kuberpult/assets/133217951/171f571c-2627-43db-b4e6-00374c042d26)
![image](https://github.com/freiheit-com/kuberpult/assets/133217951/e2092fdb-70a3-4462-a84d-65ef440fc248)
![image](https://github.com/freiheit-com/kuberpult/assets/133217951/b9539842-c325-490c-98ec-cbf4fb250b1d)


well it is not beautiful, as it is not trimmed anywhere - I'll have a look if I see some low hangig fruits to adapt in the UI

Reply:
Overall this looks ok, just the "planned actions" need to be adapted. There should not be a new scrollbar.

Reply:
There is still a scrollbar in the planned actions:
![image](https://github.com/freiheit-com/kuberpult/assets/3481382/c5c2f74a-d7f8-47af-94f5-bf4743545c0b)

Instead it should automatically add newlines. Maybe `word-wrap:break-word;` is enough to achieve that. @jdvgh

Reply:
Got it to work with `word-break: break-word`
![image](https://github.com/freiheit-com/kuberpult/assets/133217951/2353274e-0c11-46e3-b6ca-e911cde80b52)